pkg/controller/controller_instance.go

```diff
@@ -302,6 +333,33 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 	glog.V(4).Infof("Provisioning a new Instance %v/%v of ServiceClass %v at Broker %v", instance.Namespace, instance.Name, serviceClass.Name, brokerName)
 	response, err := brokerClient.ProvisionInstance(request)
 	if err != nil {
+		// There are two buckets of errors to handle:
+		// 1.  Errors that represent a failure response from the broker
+		// 2.  All other errors
+		if osb.IsHTTPError(err) {
+			// An error from the broker represents a permanent failure and
+			// should not be retried; set the Failed condition.
+
+			httpErr := err.(osb.HTTPStatusCodeError)
+			s := fmt.Sprintf("Error provisioning Instance \"%s/%s\" of ServiceClass %q at Broker %q: %s", instance.Namespace, instance.Name, serviceClass.Name, brokerName, httpErr)
```

Comment:
inconsistently quoting names in this message, its also inconsistent with another similar log message above on L333. Keep in mind that its likely this message will be presented to end users since its being used in a Condition.